packages/analytics-types/src/autocapture.ts

```diff
@@ -0,0 +1,53 @@
+import { Logger } from './logger';
+
+export type ActionType = 'click' | 'change';
+
+export interface AutocaptureOptions {
+  /**
+   * List of CSS selectors to allow auto tracking on.
+   * When provided, allow elements matching any selector to be tracked.
+   * Default is ['a', 'button', 'input', 'select', 'textarea', 'label', '[data-amp-default-track]', '.amp-default-track'].
+   */
+  cssSelectorAllowlist?: string[];
```

Comment:
Do we also want to export some constants like https://github.com/amplitude/Amplitude-TypeScript/blob/cd9ce6c12eec07501e231d5871f2ca36bf105b38/packages/plugin-autocapture-browser/src/index.ts#L4C3-L5? So user can config to override like this:
```
cssSelectorAllowlist: [...DEFAULT_CSS_SELECTOR_ALLOWLIST, ".my-class"],
```

Reply:
I added them at https://github.com/amplitude/Amplitude-TypeScript/pull/780/commits/cd72664d388e6381cfd1875224d0c9ef7e769690